app/scripts/modules/cloudfoundry/src/help/cloudfoundry.help.ts

```diff
@@ -1,6 +1,10 @@
 import { HelpContentsRegistry } from '@spinnaker/core';
 
 const helpContents: { [key: string]: string } = {
+  'cf.service.deploy.timeout':
+    '(Optional) <b>Override Deploy Timeout</b> is to override the default timeout value of 450 seconds',
```

Comment:
seems like the help content should explain the impact that value has without reusing the terms being described

Reply:
I've changed the text but I'm open for suggestions on better help text.

Reply:
I meant answering what a deploy or destroy timeout mean. I actually don't know exactly. Does it expect the deployment to happen and become healthy within a certain amount of time? If it doesn't then what happens? Does it try again, does the server group get deleted? What if it times out on a destroy? I guess it is still taking traffic? What info could help me understand what these values should be for my application?

Reply:
👍

Reply:
@claymccoy If I understand you correctly, you'd still expect to see some text that says something to the effect that "even if the operation times out, a service may yet get created and be bindable" and the opposite for destroy?

Reply:
👍 
I've added additional clarification to the help text